server/project.go

```diff
@@ -88,14 +89,24 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch project list").SetInternal(err)
 		}
 
+		activeProjectList := make([]*api.Project, 0)
 		for _, project := range list {
 			if err := s.composeProjectRelationship(ctx, project); err != nil {
 				return echo.NewHTTPError(http.StatusInternalServerError, fmt.Sprintf("Failed to fetch project relationship: %v", project.Name)).SetInternal(err)
 			}
+			// We will filter those project with the current principle as an inactive member (the role provider differs from that of the project)
+			if projectFind.PrincipalID != nil {
```

Comment:
Not sure if we can do this in findProjectList from store/project.go

id IN (SELECT project_id FROM project_member WHERE principal_id = $%d AND role_provider = project.role_provider)